src/test/javascript/spec/component/programming-exercise/programming-exercise-custom-build-plan.component.spec.ts

```
@@ -264,4 +265,10 @@ describe('ProgrammingExercise Custom Build Plan', () => {
         expect(mockAeolusService.getAeolusTemplateFile).not.toHaveBeenCalled();
         expect(comp.programmingExercise.buildConfig?.buildScript).toBe('echo "test"');
     });
+
+    it('should set timeout correctly', () => {
+        comp.programmingExercise.buildConfig!.timeoutSeconds = 100;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)